### DISCUSSION TITLE
Getting oriented

### DISCUSSION THREAD

**Discussion by ColinBroughNHS** [intersystems-community/vscode-objectscript#1024] [Q&A] [answered]
I've tried reading the documentation to the extensions, but still left floundering a bit. Looking for more introductory information:

Developing using Ensemble/Cache in a healthcare setting. Historically team used Studio against a single dev server. Now looking to move to developing locally (each of us having an Ensemble instance on our own machines), and managing code to a GitLab instance managed within the organisation. And VS Code as part of the development environment.

We've managed to get VSCode + the 3x extensions connecting to our various servers (the local instances on our own machines, and our dev server). But we are confused about **where** the files are saved locally - and how we can create local git repos for those files we can then push to our GitLab central repo. There may be things in the documentation that are clear if you already know VSCode and what it calls things, but its confusing us! 

Is there other introductory documentation anywhere apart from https://intersystems-community.github.io/vscode-objectscript/, or anything on YouTube or elsewhere? Thanks

**Comment by gjsjohnmurray** [comment, marked as answer]
Pinging @isc-rsingh

**Reply by ColinBroughNHS** [reply]
Following up my own query: I'm a bit further on, and key was this video on the InterSystems Learning site: https://learning.intersystems.com/course/view.php?id=1458 The key step for us was exporting files from server into local folder (2:10 or so in the video). But orienting to the model being used by the extensions, VS Code, and Ensemble itself is tricky - especially when terms like "export" are used at different points for separate, distinct steps.

**Reply by isc-rsingh** [reply]
Hi @ColinBroughNHS. Glad you found the answer to your question. InterSystems' Learning has a number of resources around VSCode in addition to that one. If you search for VSCODE on https://learning.intersystems.com, you'll find about 10 resources of varying length, and there's also a YouTube playlist here: https://www.youtube.com/playlist?list=PLp4xNHWZ7IQm685y7r2LMzrRYQ7ef5ZWn.
